.codeclimate.yml

```diff
@@ -1,8 +1,14 @@
 engines:
   fixme:
+    enabled: false
```

Comment:
It could be! 

Enabling seemed to require excluding the main files of the project (because they have the name `Fixme` peppered throughout), so ESLint seemed more valuable to me.

I can revert and/or think about a way to avoid false positives from Fixme